### PR TITLE
Add xadatasource support

### PR DESCRIPTION
**Pull request by falle1234Akait**

### CODE REVIEW

**Comment by guidograzioli** [comment]
Can you please rebase and send to the main branch as well?

**Comment by guidograzioli** [comment]
Can you please rebase and send to the main branch as well?

File: plugins/templates/rules/xadatasources.j2 [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{ "xadatasource" => {
+  "{{ values['name'] }}" => {
+    {% if 'pool_name' in values %} "pool-name" => "{{ values['pool_name'] }}", {% endif %}
+    "xa-datasource-properties" => {
+      {% if 'url' in values['xa-datasource_properties'] %} "URL" => { "value" => "{{ values['xa_datasource_properties']['url'] }}" } {% endif %}
```
Comment (guidograzioli) [inline plugins/templates/rules/xadatasources.j2:5]: only a typo here (`xa-datasource_properties` -> `xa_datasource_properties`), rest is LGTM!

**Review by guidograzioli — CHANGES_REQUESTED** [review]

File: plugins/templates/rules/xadatasources.j2 (hunk 1 above)
Comment (falle1234Akait) [reply plugins/templates/rules/xadatasources.j2:5]: You got sharp eyes :)
I'll fix it right away

**Review by guidograzioli — APPROVED** [review]

**Pull request merged**